src/js/NativeRNSentry.ts

```diff
@@ -46,6 +46,7 @@ export interface Spec extends TurboModule {
   initNativeReactNavigationNewFrameTracking(): Promise<void>;
   captureReplay(isHardCrash: boolean): Promise<string | undefined | null>;
   getCurrentReplayId(): string | undefined | null;
+  crashedLastRun(): Promise<boolean | null>;
```

Comment:
iOS nil is converted to undefined and Java null to null, it should not cause any issues, but just so the typing is correct.

```suggestion
  crashedLastRun(): Promise<boolean | undefined | null>;
```